Adding Warning section about Secure by Default (#127226)
Azure/azure-content: articles/virtual-network/ip-services/public-ip-upgrade.md

In this article, you learn how to upgrade a static Basic SKU public IP address to Standard SKU in the Azure portal, Azure CLI, or Azure PowerShell.

> > [!WARNING]

Reviewer: ```suggestion
> [!IMPORTANT]
```
Please change the call-out to use IMPORTANT vs. WARNING.

> > Be aware that Standard SKU IP Addresses are considered Secure by default and will be closed to all internal traffic. It is recommended to apply an NSG to the Subnet or Network Interface that the Public IP address is assoicated in order to prevent unexpected downtime, see [Public IP addresses in Azure](https://learn.microsoft.com/en-us/azure/virtual-network/ip-services/public-ip-addresses#sku)

Reviewer 2: ```suggestion
> Be aware that Standard SKU IP Addresses are considered Secure by default and will be closed to all internal traffic. It is recommended to apply an NSG to the Subnet or Network Interface that the Public IP address is assoicated in order to prevent unexpected downtime, see [Public IP addresses in Azure](https://learn.microsoft.com/azure/virtual-network/ip-services/public-ip-addresses#sku)
```
fixing build error

> > Be aware that Standard SKU IP Addresses are considered Secure by default and will be closed to all internal traffic. It is recommended to apply an NSG to the Subnet or Network Interface that the Public IP address is assoicated in order to prevent unexpected downtime, see [Public IP addresses in Azure](https://learn.microsoft.com/en-us/azure/virtual-network/ip-services/public-ip-addresses#sku)

Reviewer: ```suggestion
> Standard SKU IP Addresses are deployed using the [**Secure by default**](public-ip-addresses.md#sku) model. By default, IP addresses will be closed to all internal traffic when used as a front-end. To allow traffic and prevent unexpected downtime, apply a [network security group (NSG)](../../virtual-network/network-security-groups-overview.md#network-security-groups) to public IP address' associated Network Interface (NIC) or subnet.
```

Author: @mbender-ms I made the changes to the pull request. Please let me know if there is anything else I had missed. Thanks!
